pkg/transforms/filter.go

```diff
@@ -30,60 +30,111 @@ type Filter struct {
 	FilterOut    bool
 }
 
-// NewFilter creates, initializes and returns a new instance of Filter
-func NewFilter(filterValues []string) Filter {
-	return Filter{FilterValues: filterValues}
+// NewFilterFor creates, initializes and returns a new instance of Filter
+// that defaults FilterOut to false so it is filtering for specified values
+func NewFilterFor(filterValues []string) Filter {
```

Comment:
interesting naming here; i would prefer to call it `query` instead of `filter` since `filter` usually has the connotation to sift out things.

Reply:
@jim-wang-intel , It filters `for` only those specified. I.e. sifts out all others.